quartodoc/autosummary.py

```diff
@@ -588,7 +598,7 @@ def build(self, filter: str = "*"):
         # sidebar ----
 
         if self.sidebar:
-            _log.info(f"Writing sidebar yaml to {self.sidebar}")
+            _log.info(f"Writing sidebar yaml to {self.sidebar['file']}")
             self.write_sidebar(blueprint)
```

Comment:
Actually, I think I've convinced myself that everything is all good. Here's where we decide whether or not to write the sidebar, by which point `self.sidebar` is guaranteed to be a dictionary containing at least a `file` item (in `Builder.__init__` just above in the diff).